CveXplore/database/maintenance/Sources_process.py

```diff
@@ -1094,6 +1094,80 @@ def populate(self, **kwargs):
         return self.update()
 
 
+class EPSSDownloads(CSVFileHandler):
+    def __init__(self):
+        self.feed_type = "EPSS"
+        self.delimiter = ","
+        super().__init__(self.feed_type, self.delimiter)
+
+        self.feed_url = Configuration.getFeedURL(self.feed_type.lower())
+        self.logger = logging.getLogger(__name__)
+        self.is_update = True
+
+    @staticmethod
+    def process_epss_item(item=None):
+        if item is None:
+            return None
+
+        epss = {
```

Comment:
Additional comment; might be useful to add the epss date as well? They release on a daily schedule, right?

Reply:
Yes, they release it on a daily schedule. It sounds good, I will add the date as well.

Reply:
Awesome!